docs/core/tutorials/netcore-hosting.md

```diff
@@ -25,11 +25,11 @@ Because hosts are native applications, this tutorial will cover constructing a C
 
 You will also want a simple .NET Core application to test the host with, so you should install the [.NET Core SDK](https://www.microsoft.com/net/core) and [build a small .NET Core test app](../../csharp/getting-started/with-visual-studio.md) (such as a 'Hello World' app). The 'Hello World' app created by the new .NET Core console project template is sufficient.
 
-This tutorial and its [associated sample](https://github.com/dotnet/docs/tree/master/samples/core/hosting) build a Windows host, but please see the notes at the end of this article about hosting on Unix.
+This tutorial and its associated sample build a Windows host, but please see the notes at the end of this article about hosting on Unix.
```

Comment:
Nit: I've never liked "please see". I'd change this to "...build a Windows host; see the notes at the end..."